doc/usage/extensions/autosummary.rst

```diff
@@ -99,6 +99,12 @@ The :mod:`sphinx.ext.autosummary` extension does this in two parts:
 
      .. versionadded:: 1.0
 
+   * You can specify to documented imported classes and functions at a module
+     level using the new :confval:`autosummary_imported_members` config value.
+     By default this if disabled.
```

Comment:
This text is duplicated with `confval` entry. It would be better to merge this into the entry.